gatsby/src/pages/drugs/{BnfDrug.slug}/medicinal-forms.tsx

```diff
@@ -92,6 +160,15 @@ export const query = graphql`
 					preps {
 						...FullPrep
 					}
+					cautionaryAndAdvisoryLabels {
+						label {
+							number
+							description
```

Comment:
I don't think we're displaying the `description` here so no need to query for it, it just bloats the json static props for the page